xlson/__init__.py

```diff
@@ -53,6 +55,12 @@ def __init__(self, **kwargs: Dict) -> None:
         elements[KEY] = kwargs[NAME]
         elements[TYPE] = self.field_type
 
+        if "required" in kwargs.keys():
```

Comment:
```python
if "required" in kwargs:
    ...
````
I think this will be consistent with the codebase so far.

Reply:
Sure..changed that